autovalet_goal_generation/launch/seg_map.launch

```diff
@@ -0,0 +1,85 @@
+<?xml version="1.0"?>
```

Comment:
is this file not a lot of copy/pasted from parking_lot.launch? could we clean that up?

Reply:
Yeah. it's mostly a copy of parking_lot launch. How do you want me to clean it up?